tests/klay_blockchain_test.go

```diff
@@ -48,31 +48,40 @@ func TestSimpleBlockchain(t *testing.T) {
 
 	// create account
 	richAccount, accounts, contractAccounts := createAccount(t, numAccounts, validator)
+	time.Sleep(time.Second)
```

Comment:
This sleep is necessary?

Reply:
No it's not, I will remove this part. Thanks.